docs_src/pipelinebenchmarking.md

```diff
@@ -176,10 +171,6 @@ sudo ./benchmark.sh --stream_density <target FPS> --logdir <output dir>/data --i
 ---
 ## Appendix: Benchmark Helper Scripts
 
-- **camera-simulator.sh**
-
-Starts the camera simulator. To use, place the script in a folder named camera-simulator. At the same directory level as the camera-simulator folder, create a folder called sample-media. The camera-simulator.sh script will start a simulator for each .mp4 video that it finds in the sample-media folder and will enumerate them as camera_0, camera_1 etc.  Be sure the path to camera-simulator.sh script is correct in the camera-simulator.sh script.  
-
-- **stop_server.sh**
+- `camera-simulator.sh`: This script starts the camera simulator. Create two folders named **camera-simulator** and **sample-media**. Place `camera-simulator.sh` in the **camera-simulator** folder. At the same directory level as the camera-simulator folder, create a folder called sample-media. The `camera-simulator.sh` script will start a simulator for each *.mp4* video that it finds in the **sample-media** folder and will enumerate them as camera_0, camera_1, and so on. Make sure that the path to the `camera-simulator.sh` script is mentioned correctly in the `camera-simulator.sh` script.  
```

Comment:
The original text instructs to create a folder called sample-media, and that the script would start a simulator for each mp4 file in that folder. Does the script download the mp4 files automatically or should the user copy or download the files to the sample-media folder?

Reply:
The camera-simulator will not automatically download media. The user will either need to copy their file into sample-media or run [download_sample_videos.sh](https://github.com/intel-retail/automated-self-checkout/blob/main/benchmark-scripts/download_sample_videos.sh)